File: Exercises_Python/2486.py


t = int(input())

while t != 0:
    acumulador = 0
    for _ in range(0,t):
        entrada = input().split(maxsplit=1)  
        quantidade = int(entrada[0])
        tipo = entrada[1].strip()

        if tipo == 'suco de laranja':
            tipo = 120
            tipo = int(tipo)
        elif tipo == 'morango fresco' or tipo == 'mamao':
            tipo = 85
            tipo = int(tipo)
        elif tipo == 'goiaba vermelha':
            tipo = 70
            tipo = int(tipo)
        elif tipo == 'manga':
            tipo = 56
            tipo = int(tipo)
        elif tipo == 'laranja':
            tipo = 50
            tipo = int(tipo)
        elif tipo == 'brocolis':
            tipo = 34
            tipo = int(tipo)
        
        qnt_vitc = quantidade * tipo
        acumulador = acumulador + qnt_vitc

    if (acumulador >= 110) and (acumulador <= 130):
        print('{} mg'.format(acumulador))
    elif acumulador > 130:
        print('Menos {} mg'.format(acumulador - 130))
    elif acumulador < 110:
        print('Mais {} mg'.format(110 - acumulador))

    t = int(input())



        
        

File: Bee-Crowd/Exercises_C++/2486.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

	int qntd;
	cin >> qntd;
	int numero;
    string tipo;

	while (qntd != 0) {
		
		int soma = 0;

		for (int c=0 ; c<qntd ; c++) {
			cin >> numero;
			cin.ignore();
			getline(cin,tipo);

			if (tipo == "suco de laranja") {
				soma += (numero * 120);
			}
			else if (tipo == "morango fresco") {
				soma += (numero * 85);
			}
			else if (tipo == "mamao") {
                soma += (numero * 85);
            }
			else if (tipo == "goiaba vermelha") {
                soma += (numero * 70);
            }
			else if (tipo == "manga") {
                soma += (numero * 56);
            }
			else if (tipo == "laranja") {
                soma += (numero * 50);
            }
			else if (tipo == "brocolis") {
                soma += (numero * 34);
            }
		}

		if (soma < 110) {
			cout << "Mais " << 110-soma << " mg" << endl;
		} 
		else if (soma > 130) {
			cout << "Menos " << soma-130 << " mg" << endl;
		}
		else {
			cout << soma << " mg" << endl;
		}

		cin >> qntd;
	}
}
